### PR TITLE
[Jormungandr] monitor jormungandr

### PR DESCRIPTION
**Pull request by azime**
Jira: [NAV-2329](https://navitia.atlassian.net/browse/NAV-2329)

[NAV-2329]: https://navitia.atlassian.net/browse/NAV-2329?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docker/debian11/Dockerfile-jormungandr [hunk 1]
```diff
@@ -49,6 +49,6 @@ RUN a2ensite 000-default.conf \
 
 HEALTHCHECK CMD curl -f http://localhost/v1 || exit 1
 
-EXPOSE 80
+EXPOSE 80 9091
```
Comment (pbougue) [inline docker/debian11/Dockerfile-jormungandr]: OK, so not using apache on the new health endpoint.

File: docker/run_jormungandr.sh [hunk 2]
```diff
@@ -1,14 +1,27 @@
 #!/usr/bin/env bash
 
+app_processes=${1-1}
```
Comment (pbougue) [inline docker/run_jormungandr.sh]: So first arg, and default value is `1`.
:heart: bash

**Review by pbougue — APPROVED** [review]

File: docker/debian11/Dockerfile-jormungandr (hunk 1 above)
Comment (Aletor93) [reply docker/debian11/Dockerfile-jormungandr]: I'm asking myself if you should not also specify the port `5050` which is exposed for uwsgi stats.

File: docker/run_jormungandr.sh [hunk 3]
```diff
@@ -1,14 +1,65 @@
 #!/usr/bin/env bash
 
+file=/usr/src/app/jormungandr.wsgi
+jormungandr_cache2="name=jormungandr,items=2048"
+monito_cache2="name=monitor,items=100"
```
Comment (pbougue) [inline docker/run_jormungandr.sh]:
```suggestion
monitor_cache2="name=monitor,items=100"
```

File: docker/run_jormungandr.sh [hunk 4]
```diff
@@ -1,14 +1,65 @@
...
+
+
+function show_help() {
+    cat << EOF
+Usage: ${0##*/} -j jormungandr-processes -m monitor-processes -r max-requests
+    -j      jormungandr-processes
```
Comment (pbougue) [inline docker/run_jormungandr.sh]: below, the naming is app_processes:
```suggestion
    -a      app-processes number
```

File: docker/run_jormungandr.sh [hunk 5]
```diff
@@ -1,14 +1,65 @@
...
+    -m      [0|1] monitor-processes
```
Comment (pbougue) [inline docker/run_jormungandr.sh]:
```suggestion
    -m      [0|1] activate monitor-process
```

File: docker/run_jormungandr.sh [hunk 6]
```diff
@@ -1,14 +1,65 @@
...
+    -r      max-requests for jormungandr
```
Comment (pbougue) [inline docker/run_jormungandr.sh]:
```suggestion
    -r      max-requests before reload for jormungandr worker
```

**Review by Aletor93 — APPROVED** [review]

**Pull request merged**